archive/archive_api/src/test_bag_request.py

```diff
@@ -0,0 +1,87 @@
+# -*- encoding: utf-8 -*-
+
+import json
+
+from bag_request import (
+    bag_request
+)
+
+
+def test_looks_up_bag(dynamodb_client, dynamodb_resource, s3_client, bucket_bag, table_name_bag):
+    bag_id = 'b12345678x'
+    stored_bag = {'id': bag_id}
+
+    given_bag_in_vhs(
+        bag_id,
+        stored_bag,
+        dynamodb_client,
+        s3_client,
+        bucket_bag,
+        table_name_bag
+    )
+
+    response = bag_request(
+        dynamodb_resource,
+        table_name_bag,
+        s3_client,
+        bucket_bag,
+        bag_id
+    )
+
+    assert response == stored_bag
+
+
+def run_around_tests(dynamodb_client, s3_client, bucket_bag, table_name_bag):
```

Comment:
Where's this method used?

Reply:
@alexwlchan This was copy pasted in from the test_archive_bags.py file, but it looks like I missed the decorator `@pytest.yield_fixture(autouse=True)` so it won't be called at all. Worth adding do you think?